### DISCUSSION TITLE
仓库创建/归档申请

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [tech-podcasts/community#9] [General]
为避免创建仓库的权限过于开放，有需要创建仓库（repository）的情况，请按照如下格式回复：

* 仓库名称：xxx
* 理由：xxx
* 管理员：xxx

**Comment by le0zh0u** [comment]
仓库名称：2023TechPodFest
理由：提交活动介绍、物料等
管理员：@le0zh0u

**Reply by LinuxSuRen** [reply]
只叫 TechPodFest 是否可以呢，如果每年创建一个仓库的话，会感觉有点混乱。可以在仓库内部用文件夹的方式来区分年份。

**Reply by le0zh0u** [reply]
因为考虑的是每年的活动重点会不一样，或者我们在这建一个仓库，用来汇总活动的。

**Reply by le0zh0u** [reply]
完整的文案啥的我们放到别的地方，这样组织看起来整洁一些

**Reply by LinuxSuRen** [reply]
ok

**Comment by LinuxSuRen** [comment]
![image](https://github.com/tech-podcasts/community/assets/1450685/b1944854-3c0f-4b48-949f-9f28bdd3db36)
